src/components/Navbar.js

```diff
@@ -0,0 +1,23 @@
+import {  Link } from "react-router-dom";
+import '../App.css';
+function Navbar  ()  {
+  return (
+      <div>
+          <nav className="Navbar w-100">
+          <Link  className="text-decoration-none text-light logo" to="/">MyButiQ</Link>
+          <input type="checkbox" className="" id="toggler"/>
+          <label for="toggler" className=""><svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" fill="currentColor" class="bi bi-list" viewBox="0 0 16 16">
```

Comment:
Use SVG from a different file, it seems so messy here
![Screenshot_2022-06-11-20-30-22-747_com.android.chrome.jpg](https://user-images.githubusercontent.com/65453213/173198529-f0d87e4f-4860-4eeb-97a4-05398e275113.jpg)
You can check the image as reference